database/sproc/group.sql

```diff
@@ -54,3 +56,24 @@ END$$
 
 DELIMITER ;
 
+DELIMITER $$
+USE `doughBros_db`$$
+CREATE PROCEDURE `getGroupsByUser`(IN `user_id` INT(8))
+BEGIN 
+
+SELECT * FROM `group_membership` WHERE `did_accept_invite` = TRUE AND `fk_user_id` = `user_id`;
```

Comment:
```suggestion
SELECT * FROM `group_membership` WHERE (`did_accept_invite` = TRUE AND `fk_user_id` = `user_id`);
```
Not actually sure if this is needed but just in case